baseplate/observers/tracing.py

```diff
@@ -279,7 +279,9 @@ def _to_span_obj(
             "binaryAnnotations": binary_annotations,
         }
 
-        span["parentId"] = self.span.parent_id or 0
```

Comment:
Hm. This means we don't send a parentId at all if it's undefined. Is that right? No default?